app/main/browser.js

```diff
@@ -54,9 +55,39 @@ app.on('ready', async function() {
   // Register http content protocols, e.g. for displaying `tofino://` pages.
   protocols.registerHttpProtocols();
 
-  await BW.createBrowserWindow(userAgentClient, () => {
+  const bw = await BW.createBrowserWindow(userAgentClient, () => {
     instrument.event('browser', 'READY', 'ms', Date.now() - browserStartTime);
   });
+
+  const options = parseArgs();
+  if (options.test) {
+    bw.once('window-ready', () => {
+      const Mocha = require('mocha'); // eslint-disable-line
```

Comment:
Let's avoid polluting browser.js with inline code. This should be in an utility module or just another file.